diff --git a/CMake/vtkDetermineCompilerFlags.cmake b/CMake/vtkDetermineCompilerFlags.cmake
index db642b8..709212c 100644
--- a/CMake/vtkDetermineCompilerFlags.cmake
+++ b/CMake/vtkDetermineCompilerFlags.cmake
@@ -1,6 +1,10 @@
 SET(VTK_REQUIRED_C_FLAGS)
 SET(VTK_REQUIRED_CXX_FLAGS)
 
+# Fields inside Tcl_Interp are deprecated. Temporarily enable them
+SET(VTK_REQUIRED_CXX_FLAGS
+  "${VTK_REQUIRED_CXX_FLAGS} -DUSE_INTERP_RESULT")
+
 # make sure Crun is linked in with the native compiler, it is
 # not used by default for shared libraries and is required for
 # things like java to work.
